scripts/initialize.sh

```diff
@@ -107,14 +123,41 @@ if [ ! -z "$authorization_amount" ]; then
    authorization_amount_opt="--authorization ${authorization_amount}"
 fi
 
-cd $KEEP_BEACON_SOL_PATH
-    
 printf "${LOG_START}Setting up staking...${LOG_END}"
 
-stake="npx hardhat stake --network $NETWORK --owner ${stake_owner} \
-    --provider ${staking_provider} --operator ${operator} \
-    --beneficiary ${beneficiary} --authorizer ${authorizer} "
+mint="npx hardhat mint --network $NETWORK --owner ${stake_owner}"
+stake="npx hardhat stake --network $NETWORK --owner ${stake_owner} --provider ${staking_provider} --beneficiary ${beneficiary} --authorizer ${authorizer} "
+increase_authorization="npx hardhat increase-authorization --network $NETWORK --owner ${stake_owner} --provider ${staking_provider} --authorizer ${authorizer}"
+register_operator="npx hardhat register-operator --network $NETWORK --owner ${stake_owner} --provider ${staking_provider} --operator ${operator}"
+
+application="--application RandomBeacon"
+
+if [ "$SKIP_ECDSA_DEPLOYMENT" = true ]; then
+   cd $KEEP_BEACON_SOL_PATH
+   # go to beacon
+elif [ "$SKIP_TBTC_DEPLOYMENT" = true ]; then
+   # go to ecdsa (includes beacon contracts)
+   cd $KEEP_ECDSA_SOL_PATH
+else
+   # go to tbtc (includes beacon and ecdsa contracts)
+   if [ "$TBTC_PATH" = "" ]; then
+      cd "$KEEP_TBTC_SOL_PATH"  
+   else
+      cd "$TBTC_PATH/solidity"
+   fi
+fi
+
+eval ${mint} ${stake_amount_opt}
+eval ${stake} ${stake_amount_opt}
```

Comment:
We could have mint and stake scripts only in random beacon, right? I like this idea.

Reply:
Yes, now we have dedicated 4 tasks for random beacon: mint, stake, authorize, register-operator. And 2 tasks for ecdsa: authorize, register-operator.